### ISSUE TITLE
Show last used patterns (maybe with favorites)

### ISSUE THREAD

**Issue by Hannott** [Wolfsblvt/BlossomsPokemonGoManager#295]
When renaming, it would be nice to have some default schemes. I rename all mine with IV-att/def/sta (98-15/15/14). Would be nice to just have a %iv_stats% parameter for this.

**Comment by Wolfsblvt** [comment]
We had that, but it is very specific. Guess it is better to keep the patterns as short as possible so that you can combine them in any way you want.

What **is** possible and would be cool though is that you have a list of your last entered renames where you can easily select some, maybe even with a favorite feature so that it always stays on top of that list.
I'll keep that issue open for that.

(Btw, you can use the IV hexadecimal pattern, which has all three and even uses just three characters of the name :P )

**Comment by Hannott** [comment]
Yeah, it's very specific, but I assume a lot of people would use the same naming schemes if a few good default ones were available.

This is _very_ clunky: %iv_rating%-%iv_att%/%iv_def%/%iv_stam%

To not clutter it up, it could be added to a dropdown. Favorites would be nice, but that's more work, and I assume is lost when you get the next version. You could put it in config of course, but a lot of things end up in config after a while, and it gets cluttery

**Comment by FernandoTBarros** [comment]
I've just implemented a "show last used" pattern. Stores in config.json file and load when the panel opens.
How do I send a merge request? Do I need to open an issue? Or just commit in the developer branch?

**Comment by Hannott** [comment]
@FernandoTBarros Submit a pull request to the develop branch

**Comment by Wolfsblvt** [comment]
Yep. Make a pull request from your branch to the developer branch, then your code gets reviewed and will eventually be merged, if everything is alright.

**Comment by FernandoTBarros** [comment]
OK Done! I didn't associate this issue with the merge request. It's a good thing to do?

**Comment by ISeeTanamon** [comment]
You can mark this done @Wolfsblvt. Going in and out of Manager and back in, saves the last used pattern in field.

**Comment by FernandoTBarros** [comment]
I think he kept this opened because of the "favorite" part. It's an interesting idea have your last used patterns, or a place for organizing your renaming patterns.

**Comment by Wolfsblvt** [comment]
Yep, exactly that. I guess many will have similar things like me. I name my fighters (attackers and defenders) other than the Pokémon I just want to evolve, so I need at least two rename patterns that I will use very regularly.

Would be a cool feature for the new UI.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using BlossomsPokemonGoManager. This project is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving it. Please do not use it with a live account or current credentials. See the [Disclaimer](https://github.com/Wolfsblvt/BlossomsPokemonGoManager#disclaimer) for details.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. Blossoms Pokémon Go Manager is no longer maintained and no longer works with current Pokémon Go services, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.
